styles.css

```diff
@@ -34,34 +34,60 @@ main {
 }
 
 section {
-  border: solid 2px darkred;
+  border: solid 4px darkred;
   display: flex;
   flex-direction: column;
   width: 90%;
   margin: auto;
   padding: 20px 20px;
-  align-items: center;
   list-style: none;
 }
 
 ul {
   display: flex;
   flex-direction: row;
-  justify-items: flex-end;
+  justify-items: end;
+  align-items: flex-start;
   list-style: none;
+  padding-top: 15px;
 }
 
 .flexbox-container {
+  position: relative;
+  top: -25px;
+  right: auto;
```

Comment:
I am not sure if you need this part here. See my next comment that may answer your question too. :)